lib/travis/api/v3/renderer/user.rb

```diff
@@ -2,6 +2,28 @@
 
 module Travis::API::V3
   class Renderer::User < Renderer::Owner
-    representation(:standard, :is_syncing, :synced_at)
+    representation(:standard, :is_syncing, :synced_at, :first_logged_in_at)
+
+    def render(representation)
+      result = super
+
+      if hmac_secret_key
+        result['secure_user_hash'] = secure_user_hash
+      end
+
+      result
+    end
```

Comment:
I couldn’t find a way to conditionally include an attribute in a representation, but I saw this technique in another renderer, hopefully it’s acceptable?

Reply:
hmm it’s too bad the little expander things don’t show here because the code shown in this preview doesn’t really reflect my question 🤔